sonic_platform_base/module_base.py

```diff
@@ -258,6 +262,31 @@ def get_state_info(self):
         """
         raise NotImplementedError
 
+    def get_dpu_bus_info(self, dpu_name):
```

Comment:
@vvolam suggest to remove dpu from the API since this is base class for any Module